src/core_plugins/kibana/public/discover/controllers/discover.js

```diff
@@ -671,8 +671,11 @@ function discoverController(
       aggs: visStateAggs
     });
 
+    $scope.searchSource.onRequestStart(() => {
+      return $scope.vis.requesting();
+    });
+
```

Comment:
I think this can be shortened to `$scope.searchSource.onRequestStart($scope.vis.requesting);`?

Reply:
Calling vis.requesting without it's `this` context might work, but seems like an unnecessary risk.